README.md

```diff
@@ -179,18 +197,19 @@ jobs:
     runs-on: ubuntu-latest
     name: Deploy
     steps:
-      - uses: actions/checkout@master
+      - uses: actions/checkout@v3
       - name: Publish app
-        uses: cloudflare/wrangler-action@1.3.0
+        uses: cloudflare/wrangler-action@2.0.0
         with:
           apiToken: ${{ secrets.CF_API_TOKEN }}
+          command: publish
```

Comment:
I'd get rid of all of these from the docs to be ✨ clean ✨ if they're not required now :)